cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -1755,7 +1756,6 @@ pub(crate) fn emit(
             let (prefix, opcode) = match op {
                 SseOpcode::Movaps => (LegacyPrefixes::None, 0x0F29),
                 SseOpcode::Movapd => (LegacyPrefixes::_66, 0x0F29),
-                SseOpcode::Movdqa => (LegacyPrefixes::_66, 0x0F7F),
```

Comment:
Why was `movdqa` removed here?

Reply:
@cfallin I am actually a little confused here because I thought I introduced this during the same patch only to remove it during an amend. In any case I removed it because I questioned the correct encoding. I avoided using the instruction directly and just used the gen_move call.

About my concern on the encoding, specifically the encodings are:

`66 0F 6F /r | MOVDQA xmm1, xmm2/m128`

or 

`66 0F 7F /r | MOVDQA xmm2/m128, xmm1`

I think the former should be used, not the later. I also noticed that Movdqu also uses the later. This frankly was a todo to understand but in the moment I just used the gen_move as I noted above.

Reply:
Given the approval I'll push this assuming your fine with it still being removed.

Reply:
OK, makes sense -- in any case it's clear it's not used since the thing still compiles and works in CI!